src/test/components/__snapshots__/MarkerChart.test.js.snap

```diff
@@ -57,21 +57,21 @@ Array [
   ],
   Array [
     "fillRect",
-    288.5,
+    289,
```

Comment:
These snapshot changes come from the `Math.round` operation.

Reply:
I'm a bit surprised, I would think the new rounding logic would keep these as ending in `0.5`

Reply:
That's a good point!

This was still rounded to 1 because we set `devicePixelRatio` to `1` in tests.
I added a commit where I changed this to `2` in some tests so that this behavior is indeed tested.